### DISCUSSION TITLE
Contribution

### DISCUSSION THREAD

**Discussion by springlaughing** [code-kern-ai/refinery#276] [Q&A] [answered]
Hi there, I am interested in contributing. Do you have any preferences where I could contribute, or should I start by tackling an issue from the issue list? I have some experience in NLP (spaCy, Keras) and React.js (Typescript).

**Comment by jhoetter** [comment, marked as answer]
Hi @springlaughing , cool that you want to contribute something!

We're currently discussing which contributions there could be that are relatively easy to get started with in the backend. We're also at the moment reworking the whole UI from Angular to React. Will keep you posted :-)

Alternatively, you can also easily contribute to the content library of refinery, which is called bricks: https://github.com/code-kern-ai/bricks/issues

**Reply by springlaughing** [reply]
Sounds cool, keep me updated :) In the meantime I will take a look at bricks.
